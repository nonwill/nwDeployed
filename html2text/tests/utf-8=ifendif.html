<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<!--[if mso]><html xmlns:v="urn:schemas-microsoft-com:vml" xmlns:o="urn:schemas-microsoft-com:office:office" xmlns="http://www.w3.org/TR/REC-html40"><![endif]-->
<![if false]>
<html lang="nl">
<![endif]>

<head>
    <title>if endif</title>
    <meta name="format-detection" content="telephone=no">
    <link rel="icon" type="image/ico" href="http://www.essent.nl/favicon.ico">
    <link href="https://fonts.googleapis.com/css?family=Lato:300,400,700" rel="stylesheet">
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1">

    <!--[if lt mso 15]>
		<style>
		    span, th, td, table, div, a {
		      font-family: Arial, serif !important;
		    }
		</style>
	<![endif]-->

    <!--[if gte mso 9]>
        <xml xmlns:o="urn:schemas-microsoft-com:office:office"l>
            <o:OfficeDocumentSettings>
            <o:AllowPNG/>
            <o:PixelsPerInch>96</o:PixelsPerInch>
            </o:OfficeDocumentSettings>
        </xml>
    <![endif]-->

</head>

<body class="ioscenterfix">
    <table style="width:100%; border-collapse: collapse;">
        <tr>
            <td align="left" style="padding:0; background-color:#f1f0f0;">
				this is just a &lt;![foo]&gt; test
            </td>
        </tr>
    </table>

</html>
