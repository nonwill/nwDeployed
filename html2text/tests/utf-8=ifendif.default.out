





this is just a <![foo]> test
